python/rikai/internal/reflection.py

```diff
@@ -22,3 +22,30 @@ def find_class(class_name: str):
     module, cls = class_name.rsplit(".", 1)
     mod = importlib.import_module(module)
     return getattr(mod, cls)
+
+
+def check_class(class_name: str):
```

Comment:
even though this is internal, i would still add docstrings

Reply:
e.g., like the `class_name` should be a fully qualified name with the module path, etc